lib/trie/trie.go

```diff
@@ -865,6 +896,10 @@ func (t *Trie) deleteNodesLimit(parent *Node, limit uint32) (
 // ClearPrefix deletes all nodes in the trie for which the key contains the
 // prefix given in little Endian format.
 func (t *Trie) ClearPrefix(prefixLE []byte) {
+	defer func() {
+		t.handleTrackedDeltas(nil)
+	}()
+
```

Comment:
```suggestion
	defer t.handleTrackedDeltas(nil)

```

Reply:
Keeping it as is (so I don't forget like in a few hours on another branch) due to https://github.com/ChainSafe/gossamer/pull/2876#discussion_r989956510